src/types.ts

```diff
@@ -0,0 +1,18 @@
+export type ReviewersAll = "all";
+
+export interface Label {
```

Comment:
IMO same "@octokit/openapi-types" mentioned above could be used for all github types.